socialdistribution/templates/posts/posts_base.html

```diff
@@ -9,7 +9,7 @@
         <div class="post-heading">
             <div class="post-author">
                 <div class="post-author-img">
-                    <img src="{% static "images/temp.jpg" %}"  alt="">
+                    <img src="/media/{{ post.author.profile_img }}"  alt="">
```

Comment:
I wonder all those image file will store locally or in our database?
post.author.profile_img will be identified by the author_id?

Reply:
It will be stored in server's /media folder. The db will have the store the path that links to the image. post.author is an object somehow, so we can get the profile_img from post.author.